forecasters/animalia/R/featurizers.R

```diff
@@ -0,0 +1,40 @@
+#' Get weekly averages of daily data
+#'
+#' @param df a "wide" data frame as created by [covidcast::aggregate_signals()].
+#'   should contain the columns `geo_value` and `time_value` as well as columns
+#'   containing leads and lags. These will all begin with the string "value"
+#'
+#' @return a data frame with the same columns, but possibly fewer rows
+#' @export
+#' @importFrom dplyr group_by arrange slice_max ungroup summarise across
+#' @importFrom rlang .data
+states_featurizer <- function(df) {
```

Comment:
I think we should rename this function and the one below to reflect what it is doing rather than which task we intend to use it for.